src/Components/SavedSearchAlert/Mutations/createAdvisoryOpportunity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  createAdvisoryOpportunityMutation$data,
+  createAdvisoryOpportunityMutation,
+} from "__generated__/createAdvisoryOpportunityMutation.graphql"
+import { commitMutation, Environment, graphql } from "relay-runtime"
+
+interface MutationInput {
+  searchCriteriaID: string
+  message: string
+}
+
+export const createAdvisoryOpportunity = (
```

Comment:
Can we use our canonical useMutation pattern as seen all about?

Reply:
For this one I copied the existing `createSavedSearch` mutation as this one is linked to it - https://github.com/artsy/force/blob/f3079c38eb02009ceb4bb73f92f18bfde52782c8/src/Components/SavedSearchAlert/Mutations/createSavedSearchAlert.ts

That's how that one was setup!

Reply:
Yah, this one! similar to forque: https://github.com/artsy/force/blob/main/src/Apps/Settings/Routes/Shipping/useAddAddress.ts

Reply:
Yup - was good and added error handling!